ghost/core/core/shared/config/defaults.json

```diff
@@ -140,6 +140,9 @@
         },
         "contentAPI": {
             "maxAge": 0
+        },
+        "comments": {
```

Comment:
I do like the `contentAPI` convention used above. This is an API we are caching, so maybe `commentsAPI` would be a better fit. Also, with the discussion on the issue itself _if_ we end up only caching the comments endpoint long-term, than `commentsCountAPI` would be even better name.

Reply:
Renamed it to `commentsCountAPI`